definitions/aircraft/Asobo_CJ4.yaml

```diff
@@ -49,6 +49,11 @@ shared:
     var_name: A:ENG ANTI ICE:2
     var_units: Bool
     event_name: ANTI_ICE_TOGGLE_ENG2
+  - # ASOBO_DEICE_Switch_Pitot_Template
+    type: ToggleSwitch
+    var_name: A:PITOT HEAT
+    var_units: Bool
+    event_name: PITOT_HEAT_TOGGLE
```

Comment:
There are two pitot switches and both are already defined.

Reply:
From testing, this def is required for syncing the pitot heat. If you directly set only the lvars, the switches do not move.